epam-promo/components/overlays/NotificationCard.tsx

```diff
@@ -1,21 +1,25 @@
-import { withMods } from '@epam/uui-core';
+import { createSkinComponent, devLogger } from '@epam/uui-core';
 import * as uui from '@epam/uui';
 import { EpamPrimaryColor } from '../types';
 
 interface NotificationCardMods {
     /**
-     * Defines component color.
```

Comment:
Why did you delete the point here when you add it in other descriptions?

Reply:
fixed